backend/common/src/main/java/ai/verta/modeldb/common/config/ArtifactStoreConfig.java

```diff
@@ -1,40 +1,56 @@
 package ai.verta.modeldb.common.config;
 
+import ai.verta.modeldb.common.CommonMessages;
 import ai.verta.modeldb.common.exceptions.ModelDBException;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.rpc.Code;
-
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter(AccessLevel.PRIVATE)
```

Comment:
can we get away with `AccessLevel.NONE` ?

Reply:
Updated code with `AccessLevel.NONE`.